ui/jasmine_test/spec/helpers/jsdom_init.js

```diff
@@ -0,0 +1,14 @@
+// var jsdom = require("jsdom");
+import {JSDOM} from 'jsdom';
+import {StubMediaRecorder} from '../../../src/StubMediaRecorder.js';
```

Comment:
I'd suggest moving the StubMediaRecorder.js file into a spec/helpers/stubs folder

Reply:
+1